src/subscript/field_statistics/field_statistics.py

```diff
@@ -330,20 +329,20 @@ def main():
 def field_stat(args):
     # parse the config file for this script
     if not Path(args.configfile).exists():
-        sys.exit("No such file:" + args.configfile)
+        raise FileNotFoundError("No such file: " + args.configfile)
 
     config_file = args.configfile
     config_dict = read_field_stat_config(config_file)
     field_stat = config_dict["field_stat"]
 
     # Path to FMU project models ert/model directory (ordinary CONFIG PATH in ERT)
     if not Path(args.ertconfigpath).exists():
-        sys.exit("No such file:" + args.ertconfigpath)
+        raise FileNotFoundError("No such file: " + args.ertconfigpath)
```

Comment:
Change to f-string: `raise FileNotFoundError(f"No such file: {args.ertconfigpath}")`
And same for the other two